examples/testapp/src/pages/import-sub-account/components/DeploySubAccount.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Button } from "@chakra-ui/react";
+import { createCoinbaseWalletSDK } from "@coinbase/wallet-sdk";
+import { useCallback, useState } from "react";
+import { Client, createPublicClient, http } from "viem";
+import { createBundlerClient, createPaymasterClient, SmartAccount } from "viem/account-abstraction";
+import { baseSepolia } from "viem/chains";
+
+export function DeploySubAccount({
+  sdk,
+  subAccount,
+}: {
+  sdk: ReturnType<typeof createCoinbaseWalletSDK>;
+  subAccount: SmartAccount;
+}) {
+  const [state, setState] = useState<string>();
+
+  const handleDeploySubAccount = useCallback(async () => {
+    if (!sdk) {
+      return;
+    }
+
+    try {
+      const client = createPublicClient({
+        chain: baseSepolia,
+        transport: http(),
+      });
+      const paymasterClient = createPaymasterClient({ 
+        transport: http('https://api.developer.coinbase.com/rpc/v1/base-sepolia/S-fOd2n2Oi4fl4e1Crm83XeDXZ7tkg8O'), 
+      }) 
+      const bundlerClient = createBundlerClient({
+        account: subAccount,
+        client: client as Client,
+        transport: http("https://api.developer.coinbase.com/rpc/v1/base-sepolia/S-fOd2n2Oi4fl4e1Crm83XeDXZ7tkg8O"),
+        paymaster: paymasterClient,
+      });
+      // @ts-expect-error
+      const hash = await bundlerClient.sendUserOperation({
+        calls: [],
+      });
```

Comment:
does this actually work?

Reply:
confirmed we have empty calls test case across playground and it works